Mathlib/LinearAlgebra/AffineSpace/AffineEquiv.lean

```diff
@@ -15,46 +15,49 @@ import Mathlib.Algebra.Invertible
 /-!
 # Affine equivalences
 
-In this file we define `affine_equiv k P₁ P₂` (notation: `P₁ ≃ᵃ[k] P₂`) to be the type of affine
+In this file we define `AffineEquiv k P₁ P₂` (notation: `P₁ ≃ᵃ[k] P₂`) to be the type of affine
 equivalences between `P₁` and `P₂, i.e., equivalences such that both forward and inverse maps are
 affine maps.
 
 We define the following equivalences:
 
-* `affine_equiv.refl k P`: the identity map as an `affine_equiv`;
+* `AffineEquiv.refl k P`: the identity map as an `AffineEquiv`;
 
-* `e.symm`: the inverse map of an `affine_equiv` as an `affine_equiv`;
+* `e.symm`: the inverse map of an `AffineEquiv` as an `AffineEquiv`;
 
-* `e.trans e'`: composition of two `affine_equiv`s; note that the order follows `mathlib`'s
-  `category_theory` convention (apply `e`, then `e'`), not the convention used in function
+* `e.trans e'`: composition of two `AffineEquiv`s; note that the order follows `mathlib`'s
+  `CategoryTheory` convention (apply `e`, then `e'`), not the convention used in function
   composition and compositions of bundled morphisms.
 
-We equip `affine_equiv k P P` with a `group` structure with multiplication corresponding to
-composition in `affine_equiv.group`.
+We equip `AffineEquiv k P P` with a `Group` structure with multiplication corresponding to
+composition in `AffineEquiv.group`.
 
 ## Tags
 
 affine space, affine equivalence
 -/
 
-
 open Function Set
 
 open Affine
 
+-- Porting note: Lean4#2074
+attribute [-instance] Ring.toNonAssocRing
+
 /-- An affine equivalence is an equivalence between affine spaces such that both forward
 and inverse maps are affine.
 
-We define it using an `equiv` for the map and a `linear_equiv` for the linear part in order
+We define it using an `Equiv` for the map and a `LinearEquiv` for the linear part in order
 to allow affine equivalences with good definitional equalities. -/
-@[nolint has_nonempty_instance]
+--@[nolint has_nonempty_instance]
 structure AffineEquiv (k P₁ P₂ : Type _) {V₁ V₂ : Type _} [Ring k] [AddCommGroup V₁] [Module k V₁]
   [AddTorsor V₁ P₁] [AddCommGroup V₂] [Module k V₂] [AddTorsor V₂ P₂] extends P₁ ≃ P₂ where
   linear : V₁ ≃ₗ[k] V₂
-  map_vadd' : ∀ (p : P₁) (v : V₁), to_equiv (v +ᵥ p) = linear v +ᵥ to_equiv p
+  map_vadd' : ∀ (p : P₁) (v : V₁), toEquiv (v +ᵥ p) = linear v +ᵥ toEquiv p
 #align affine_equiv AffineEquiv
 
--- mathport name: «expr ≃ᵃ[ ] »
```

Comment:
@digama0: are we supposed to leave these lines in?

Reply:
I was told by other people (I forgot who it was) to delete these lines.